doc/source/tutorials/federated_learning/source.ipynb

```diff
@@ -0,0 +1,2614 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "146d846a-594c-4024-bb43-b04575880e67",
+   "metadata": {},
+   "source": [
+    "# Improving Chest X-ray Pneumonia Detection with Federated Learning and Covalent\n",
+    "\n",
+    "![alt text](assets/hero.png \"Covalent\")\n",
+    "\n",
+    "In this tutorial, we will provide a comprehensive guide on federated learning, dive into the technical aspects required for building Covalent workflows, and showcase how the two can be combined to build powerful experiments. The exemplar case we will employ involves the identification of pneumonia from chest X-rays."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "400f9bb8-498d-4915-ab45-d70511ef3c52",
+   "metadata": {},
+   "source": [
+    "This tutorial requires PyTorch, Hugging Face Transformers and Covalent plugins. To install all of them, simply use the requirements.txt file to replicate this notebook"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 27,
+   "id": "c1f42d9a-f340-42f3-a7fe-fd8c11d9d65d",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "covalent==0.220.0\n",
+      "covalent-awsbatch-plugin==0.26.0\n",
+      "covalent-azurebatch-plugin==0.12.0\n",
+      "covalent-gcpbatch-plugin==0.9.0\n",
+      "datasets==2.14.0\n",
+      "torch==2.0.1\n",
+      "torchvision==0.15.2\n",
+      "pandas\n",
+      "plotly\n"
+     ]
+    }
+   ],
+   "source": [
+    "with open(\"./requirements.txt\", \"r\") as file:\n",
+    "    for line in file:\n",
+    "        print(line.rstrip())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 28,
+   "id": "167485a1-d709-4f27-b4f4-6b2e6c98947e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Uncomment below line to install necessary libraries\n",
+    "# !pip install -r requirements.txt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2872e864-d3a9-4265-bece-001baa88829a",
+   "metadata": {},
+   "source": [
+    "# Federated Learning on Pneumonia Chest X-rays\n",
+    "In our demonstration of federated learning, we will train three separate Convolutional Neural Networks (CNNs) using distinct Pneumonia Chest X-ray datasets. Subsequently, the trained models and their corresponding weights will be transmitted back to a central node. On this central node, we will perform weight aggregation using a simple weighted arithmetic mean:\n",
+    "\n",
+    "$$\n",
+    "\\begin{equation}\n",
+    "w_{j} = \\sum_{k=1}^{K} \\frac{w_{jk} d_k}{\\sum_{i=1}^{K}d_i}\n",
+    "\\tag{1}\n",
+    "\\end{equation}\n",
+    "$$"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "25b99a4c-2d63-4363-9035-9a0679110901",
+   "metadata": {},
+   "source": [
+    "Here, wⱼ represents the set of weights in CNN layer j, K is the number of models/datasets, which, in our case, is three, and dₖ represents the size of the dataset k."
```

Comment:
@FilipBolt lets also use latex for the math elements in this cell.

This should do it
```
Here, $w_j$ represents the set of weights in CNN layer $j$, $K$ is the number of models/datasets, which in our case is three, and $d_k$ represents the size of the dataset $k$.
```